workflows/data_pipelines/elasticsearch/data_enrichment.py

```diff
@@ -165,6 +166,24 @@ def label_region_from_departement(departement):
     return None
 
 
+# EPCI
+def label_epci_from_commune(commune):
+    if not commune:
+        return None
+    # Modify commune if it's an arrondissement
+    commune_arrondissement_mapping = {
+        "751": "75056",  # Paris
+        "132": "13055",  # Marseille
+        "693": "69123",  # Lyon
+    }
+    for prefix, corresponding_commune in commune_arrondissement_mapping.items():
+        if commune.startswith(prefix):
+            commune = corresponding_commune
+            break
```

Comment:
Is there an EPCI for Paris, Marseille and Lyon ?
Is there always an EPCI ?

Reply:
Oh yes https://www.insee.fr/fr/metadonnees/cog/intercommunalite-metropole/EPCI200054781-metropole-du-grand-paris